File: 4344.py
N = int(input())
cnt = 0
sum = 0 
for i in range(N):
        N_List = list(map(int,input().split()))
        for j in range(1 ,len(N_List)):
            sum = sum + N_List[j]

        avg = sum / (len(N_List) -1)

        for j in range(1 ,len(N_List)):
            if N_List[j] > avg :
                cnt = cnt + 1

        print( "%.3f" %(  (cnt/N_List[0]) *100  ) ,end="" )
        print("%")

        cnt = 0
        sum = 0



 

File: 4344.c
#include <stdio.h>
#include <stdlib.h> 
#include <string.h>
#define _crt_secure_no_warnings
#pragma warning(disable:4996)

int main() {

	int C;
	int N;
	int sum = 0;
	int score;
	int save[1001] = { 0, };
	double cnt = 0;
	double avg;
	scanf("%d", &C);
	for (int i = 0; i < C; i++) {
		sum = 0;
		cnt = 0;
		scanf("%d", &N);
		if (N == 1) {
			scanf("%d", &score);
			printf("100.000%%");
		}
		else {
			for (int j = 0; j < N; j++) {
				scanf("%d", &score);
				save[j] = score;
				sum += score;
			}
			avg = (double)sum / N;
			for (int j = 0; j < N; j++) {
				if (save[j] > avg) {
					cnt++;
				}
			}
			printf("%.3lf%%\n", (cnt / N) * 100);
		}
	}

	return 0;
}